saleor/graphql/account/tests/test_account.py

```diff
@@ -5802,6 +5802,30 @@ def test_query_customer_members_with_filter_search(
     assert len(users) == count
 
 
+def test_query_customers_with_filter_by_ids(
+    query_customer_with_filter,
+    staff_api_client,
+    permission_manage_users,
+    customer_user,
+):
+    # given
+    variables = {
+        "filter": {
+            "ids": [graphene.Node.to_global_id("User", customer_user.pk)],
```

Comment:
IMO, we should add at least 3 more tests:
- with providing multiple `ids` instead of only one `id`,
- with providing empty `ids` list
- with providing don't existing customer ids